src/object/schema_system_catalog_install.cpp

```diff
@@ -452,25 +500,25 @@ namespace cubschema
     },
 // initializer
     nullptr
-    );
+	   );
+
 
-    return sm_define_domain;
   }
 
-  const system_catalog_definition &
+  system_catalog_definition
   system_catalog_initializer::get_method ()
   {
 
-    static system_catalog_definition sm_define_method (
-	    // name
-	    CT_METHOD_NAME,
-	    // columns
+    return system_catalog_definition (
+		   // name
+		   CT_METHOD_NAME,
+		   // columns
```

Comment:
It would be a good idea to adjust the indent for the first argument of system_catalog_definition(). It's indented too much.
t would look better if you aligned it with the comment line " // constraints " at the bottom.